### ISSUE TITLE
Undefined offset errors on table structure pages - master

### ISSUE THREAD

**Issue by pauluse** [phpmyadmin/phpmyadmin#11754]
As soon as I browse to the stucture page of phpMyAdmin@master it throws a few notice errors;

```
Notice in ./libraries/tbl_common.inc.php#54
 Undefined offset: 2

Backtrace

./libraries/controllers/table/TableStructureController.php#320: include_once(./libraries/tbl_common.inc.php)
./tbl_structure.php#49: PMA\libraries\controllers\table\TableStructureController->indexAction()
```

And:
```
Notice in ./libraries/tbl_common.inc.php#57
 Undefined offset: 2

Backtrace

./libraries/controllers/table/TableStructureController.php#320: include_once(./libraries/tbl_common.inc.php)
./tbl_structure.php#49: PMA\libraries\controllers\table\TableStructureController->indexAction()
```

This happens with error_reporting set to *E_ALL & ~E_DEPRECATED* which is recommended for production value.

**Comment by lem9** [comment]
I don't see that; my master is at 52f1cabcecda239095878ca02986b78d966b5b9c.

**Comment by pauluse** [comment]
Happens for me at 52f1cab as well, I'm running on nginx with php5-fpm if that matters;
nginx/1.4.6
Database client version: libmysql - 5.5.46
PHP extension: mysqli
PHP version: 5.5.9-1ubuntu4.14
Server version: 5.5.46-0ubuntu0.14.04.2-log - (Ubuntu)

I might do a little debugging of the matter tomorrow to see what is going on exactly, might be the combination with nginx or something since I tend to get 500 errors from time to time as well. Doesn't happen on the STABLE branch though.

**Comment by pauluse** [comment]
A update on the error which I'm getting, it seems that phpMyAdmin want's to be placed in a sub-folder according to the code on the line where the error occurs:

```
/**
 * Ensures the database and the table exist (else move to the "parent" script)
 * Skip test if we are exporting as we can't tell whether a table name is an alias (which would fail the test).
 */
$URL = explode('/',$_SERVER['REQUEST_URI']);
if ($URL[2])
        $requested_function = explode('?',$URL[2])[0];

if (!$URL[2] || $requested_function != 'tbl_export.php')
        require_once './libraries/db_table_exists.lib.php';
```

In my case my phpMyAdmin is directly located in the root of the domain I'm requesting. This problem doesn't happen on the STABLE branch (yet).

**Comment by nijel** [comment]
The code has been introduced in  4b8983028a5c1168a4c605070dc5431d086e7df8 as fix for #11679.
